index.js

```diff
@@ -0,0 +1,163 @@
+'use strict';
+const mongo = require('mongodb');
+const async = require('async');
+const _ = require('lodash');
+const automap = require('automap');
+const defaults = {
+  mongo: {
+    host: 'mongodb://localhost:27017',
+    collectionName: 'hapi-activities'
+  },
+  timeout: 30 * 1000, // max time an action can take, default is 30 secs, set to false for infinity
+  interval: 5 * 60 * 1000, // 5 minutes
+  log: false
+};
+
+exports.register = (server, options, next) => {
+  const settings = _.defaults(options, defaults);
```

Comment:
FYI can use `Object.assign()`, supported in node4+.

Would look something like `const settings = Object.assign({}, options, defaults);`